--- a/specs/Lispish.spec	2026-09-20 09:47:15
+++ b/specs/Lispish.spec	2026-09-20 10:10:51
@@ -66,9 +66,9 @@
 
 sbrackets: /(\[(?:[^\[\]]++|(?R))+\])/     I.return(hash("type", "SBRACKETS", "content", entry_text()))
 
-dquotes: /"(.*?)(?<!\\)"/     I.return(hash("type", "DQUOTES", "content", entry_group(0)))
+dquotes: /(?s)"(.*?)(?<!\\)"/     I.return(hash("type", "DQUOTES", "content", entry_group(0)))
 
-squotes: /'(.*?)(?<!\\)'/     I.return(hash("type", "SQUOTES", "content", entry_group(0)))
+squotes: /(?s)'(.*?)(?<!\\)'/     I.return(hash("type", "SQUOTES", "content", entry_group(0)))
 
 curlyb: /(?<!\\)\{/ /(?<!\\)\}/ I {content = undef}
  -> curlyb
